client/resource_group/controller/model.go

```diff
@@ -133,9 +138,11 @@ func newSQLCalculator(cfg *Config) *SQLCalculator {
 	return &SQLCalculator{Config: cfg}
 }
 
-// Trickle ...
-// TODO: calculate the SQL CPU cost and related resource consumption.
+// Trickle Update Sql Layer CPU consumption.
 func (dsc *SQLCalculator) Trickle(ctx context.Context, consumption *rmpb.Consumption) {
+	delta := getSQLProcessCPUTime() - consumption.SqlLayerCpuTimeMs
+	consumption.SqlLayerCpuTimeMs = delta
```

Comment:
I think it's an accumulative value, not a delta value. we can calculate the delta within `collectRequestAndConsumption`? ptal @JmPotato

Reply:
Yes, this will eventually be calculated in `collectRequestAndConsumption`. 
The reason why delta is calculated here is because it will be added here. 
https://github.com/tikv/pd/pull/6003/files#diff-ac1c6525251add035bc46a2bce40ac9d39ee05a8f1b9c062657e869d10506497R499